ui/scripts/system.js

```diff
@@ -9099,8 +9102,25 @@
                                         url: createURL('listClusters'),
                                         data: data,
                                         success: function (json) {
+                                         var clustersDetail = json.listclustersresponse.cluster;
+                                            if(clustersDetail instanceof Array){
+                                              for(var idx=0;idx< clustersDetail.length;idx++){
+                                                if (clustersDetail[idx].managedstate == 'Unmanaged') {
+                                                    clustersDetail[idx].allocationstate = 'Unmanaged';
+                                                    continue;
+                                                }
+                                                if (clustersDetail[idx].managedstate == 'Managed' && clustersDetail[idx].allocationstate == 'Enabled') {
+                                                    clustersDetail[idx].allocationstate = 'Enabled';
```

Comment:
Here we are checking if `clustersDetail[idx].allocationstate == 'Enabled'` and assigning the same value. Is it required?